.github/workflows/multi-arch-build.yaml

```diff
@@ -62,26 +62,37 @@ jobs:
           echo "$VERSION_OUTPUT"
           echo ::set-output name=version_output::$VERSION_OUTPUT
 
-      # Generate image related info - names, labels
+      # Generate image related info - names, labels, check whether to push
       - name: Generate image information
         id: image_info
         run: |
-          # have special environment variable for image name.
+          # Have special environment variable for image name.
           # Names are different for upstream and stable Buildah images
           if [ "${{ matrix.source }}" == "upstream" ]; then
             # quay.io/buildah/upstream:master
```

Comment:
Please remove master and change to "main" throughout, unless @rhatdan has a better word.

Reply:
yep, sorry, forgot to update the comment, will do.